pkg/services/serviceaccounts/database/database.go

```diff
@@ -415,6 +415,11 @@ func (s *ServiceAccountsStoreImpl) SearchOrgServiceAccounts(
 	return searchResult, nil
 }
 
+// AddServiceAccountToTeam adds a service account to a team
+func (s *ServiceAccountsStoreImpl) AddServiceAccountToTeam(ctx context.Context, serviceAccountID int64, cmd *serviceaccounts.AddServiceAccountToTeamCommand) error {
+	return s.sqlStore.AddTeamMember(serviceAccountID, cmd.OrgId, cmd.TeamId, false, models.PERMISSION_ADMIN)
```

Comment:
should we instead have an implementation of adding a service account to a team outside of `sqlStore` and make it part of the `serviceaccounts.Store` package instead? 🤔

Reply:
I think it's good as it is. It has an interface, but we know for now the implementation is the same for users & service accounts.